xml.go

```diff
@@ -301,6 +337,7 @@ type Group struct {
 	Name                    string        `xml:"Name"`
 	Notes                   string        `xml:"Notes"`
 	IconID                  int64         `xml:"IconID"`
+	CustomIconUUID          UUID          `xml:"CustomIconUUID"`
```

Comment:
It is the same as for entries:
I no custom icon is defined, the UUID field is not there at all and needs to be set as optional